docker-compose.yml

```diff
@@ -19,6 +19,7 @@ services:
     labels:
       io.balena.features.sysfs: '1'
       io.balena.features.dbus: '1'
+      io.balena.features.supervisor-api: '1'
```

Comment:
Not sure how I feel about this. Exposing the supervisor API allows you to basically fully take over the device. We might need to tighten up the security first or what do you think @robputt?

Reply:
@vpetersson we could do it through the gateway-config container instead which already has the supervisor api enabled and isn't exposed externally?

Don't think there's any other way to do it without using the API?

Reply:
Yes, doing this through the `gateway-config` container would be preferable from a security perspective as it's not publicly exposed. That said, I don't know why `gateway-config` would require access to Balena's Supervisor either. That's another component that we are yet to audit/rewrite.

Reply:
Either way, I think this is pretty essential. So one way or another we have to use the supervisor API to enable this.